_posts/2017-11-03-suds.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: SUDS
+date: 2017-11-2
+categories:
+  - projects
+description: Shower Use Detection System
+image: https://csh.rit.edu/~bigdata/SUDS.jpg
+image-sm: https://csh.rit.edu/~bigdata/SUDS.jpg
+author: Seth Gower
+author-image: https://avatars3.githubusercontent.com/u/19225089
+author-bio: First year Computer Engineering student
+author-email: seth@sethgower.com
+author-social:
+  github: https://github.com/SethGower
+  linkedin: https://www.linkedin.com/in/sethgower/
+---
+[SUDS](https://github.com/sethgower/suds), or the Shower Use Detection System, shows the status of the showers in the specific bathroom (and potentially the toilet stalls later on). When someone is in the shower and the door is locked, an LED on a panel in the hall next to the bathroom illuminates. In addition to the LEDs outside of the bathroom, I will be creating a web app to track live status updates remotely, and potentially collect statistics on when the showers/stalls are most used.
```

Comment:
>shows the status of the showers in the specific bathroom.

Where? Broaden this some more to mention that they're on floor, (i.e. where CSH is located).